### ISSUE TITLE
Why does LightBulb take so much Cpu when in options?

### ISSUE THREAD

**Issue by Antosser** [Tyrrrz/LightBulb#197]
![image](https://user-images.githubusercontent.com/71790328/111920312-8b2e7980-8a8e-11eb-9eca-cb3daf815b9c.png)

**Comment by Tyrrrz** [comment]
That looks pretty abnormal. Did it also start happening with the latest update for you?

**Comment by Tyrrrz** [comment]
Probably related to #198

**Comment by Tyrrrz** [comment]
@Antosser does this happen only when in settings?

**Comment by Antosser** [comment]
> @Antosser does this happen only when in settings?

- Yes, when I click on a slider

**Comment by Antosser** [comment]
> That looks pretty abnormal. Did it also start happening with the latest update for you?
I don't know. I've not been using LightBulb that long

**Comment by wadbr** [comment]
I'm getting 2/4% cpu usage constantly when app is in the tray and even disabled. Don't know if this is related. (Using latest portable version)

**Comment by Tyrrrz** [comment]
@wadbr probably unrelated. What's your CPU, just to have a point of reference for what 4% means.

**Comment by wadbr** [comment]
i5 4590 with an (oldish) amd gpu

I tried a couple older portable versions and it seems like the same thing exists 2.2.0, should I open a new issue?

**Comment by Tyrrrz** [comment]
It could be covered by #182 already.

I'm using i5 4460 and, as of writing this, LightBulb uses 00 CPU according to Task Manager:

![image](https://user-images.githubusercontent.com/1935960/121430977-a8f0d980-c92d-11eb-83b2-2b71e1bdb533.png)

**Comment by wadbr** [comment]
Yes the problem I'm having is constant whether the window is open or not
![image](https://user-images.githubusercontent.com/6258941/121512362-891aed80-c9e1-11eb-967f-e98e5e3a81ac.png)
Even when the app is disabled. It does seem CPU usage does not even change when I close the window or when I have it on
![image](https://user-images.githubusercontent.com/6258941/121512437-9f28ae00-c9e1-11eb-903c-6c86c69efd37.png)

I mean honestly it's not that big of a deal but it makes no sense, it should not do this.

**Comment by Tyrrrz** [comment]
Yeah :/
Unfortunately it's hard to diagnose this properly, apart from just profiling and improving performance in general, which is covered by #182 already.

**Comment by Tyrrrz** [comment]
I've come to realization that it's just the way Windows GUIs are. If you want to have some fun, try opening literally any application, track their CPU usage in Task Manager, and then move your mouse erratically on top of the application's window. CPU usage will spike significantly.

There are some separate performance challenges in the app even when it's hidden to tray (covered in #182), but they're most likely inherent to the framework the app is using unfortunately. I doubt I will have the mental capacity to find a solution to that, at least not without help. In any case, that's a separate issue, so I'll close this one based on the first paragraph.
